.rubocop.yml

```diff
@@ -2,7 +2,3 @@ inherit_gem:
   rubocop-govuk:
     - "config/default.yml"
     - "config/rails.yml"
-
-inherit_mode:
-  merge:
-    - Exclude
```

Comment:
This shouldn't have an impact on new files - it should impact how rules merge together file exclusions.

I think it's standard across GOV.UK - see: https://docs.publishing.service.gov.uk/manual/configure-linting.html

Reply:
Hmm you're right 🤔 not sure why our codebase wasn't being fully linted. I'll revert this particular change, but keep the linting fixes.

Reply:
Great stuff 👍